src/main/java/games/strategy/engine/lobby/server/UserManager.java

```diff
@@ -36,6 +36,9 @@ public String updateUser(final String userName, final String emailAddress, final
     }
     final HashedPassword password = new HashedPassword(hashedPassword);
     if (!password.isValidSyntax()) {
+      //TODO: encrypt hashedPassword and pass it to updateUser
```

Comment:
hashed password is already encrypted, it's part of the point of it not being a String   : )